### PR TITLE
Remove project root

### PR DESCRIPTION
**Pull request by jasonLaster**

### CODE REVIEW

File: src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css [hunk 1]
```diff
@@ -188,66 +161,3 @@
...
-
-.source-outline-panel.has-root > div {
-  height: 100%;
-}
-
-.source-outline-panel.has-root .thread-header {
-  margin-top: 4px;
-}
-
-.sources-list .img.blackBox {
```
Comment (jaril) [inline src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css:200]: Do we not have the blackbox feature anymore?

File: src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.js [hunk 2]
```diff
@@ -101,32 +101,7 @@ class SourceTreeItem extends Component {
       }
     }
 
-    if (isDirectory(item)) {
-      this.addCollapseExpandAllOptions(menuOptions, item);
```
Comment (jaril) [inline src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.js:105]: Don't we still need the collapse all/expand all context menu item?

If not we can take the method out as well:

https://github.com/RecordReplay/devtools/blob/74d3a8d8a2f0fab17ed37d36720c7efbcee3cd50/src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.js#L148-L164

File: src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css (hunk 1 above)
Comment (jasonLaster) [reply src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css:200]: good call we should keep this

File: src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.js (hunk 2 above)
Comment (jasonLaster) [reply src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.js:105]: oops. good catch

**Comment by jaril** [comment]
It looked like the permalink was being added to the diff 😬 Might have been a github bug.

Just needs this change: https://gist.github.com/jaril/f7869af7e591db719bc8b9ccbea12b06

**Pull request merged**